### PR TITLE
Update revista-ladinia.csl

### PR DESCRIPTION
**Pull request by schtailmuel**
page-range-format and container-title between double-quotes

### CODE REVIEW

**Comment by bwiernik** [comment]
Thanks! If you need to make additional changes, please edit the file in this pull request. Do not close it and open a new one. If you aren't sure how to do that, please ask

**Comment by schtailmuel** [comment]
ok, Thanks! I closed the previous pull request because of merge conflicts that I could not resolve.

**Comment by bwiernik** [comment]
> Hancké, Bob/Rhodes, Martin/Thatcher, Mark (eds.): Beyond varieties of capitalism: Conflict, contradiction, and complementarities in the European economy, in: , Oxford and New York 2007.

it looks like there is some issues with `book` citations. Could you take a look at that? I can help with troubleshooting if needed

**Comment by schtailmuel** [comment]
> > Hancké, Bob/Rhodes, Martin/Thatcher, Mark (eds.): Beyond varieties of capitalism: Conflict, contradiction, and complementarities in the European economy, in: , Oxford and New York 2007.
> 
> it looks like there is some issues with `book` citations. Could you take a look at that? I can help with troubleshooting if needed

Thanks for noticing the error.
I just checked, I don't see these errors in the Zotero Style Editor therefore I have trouble solving the problem.
I guess I need a check in the `container` macro, to avoid the `ìn:` being printed when there is no container-title?

I would be happy if you could help

**Comment by adam3smith** [comment]
The "in" shouldn't be there anymore now, that's a problem with the preview, but I do think the current version of the style will look wrong for a book that has an author and an editor (e.g. a classic work with an editor). How should such a citation look for the journal?

**Comment by schtailmuel** [comment]
> The "in" shouldn't be there anymore now, that's a problem with the preview, but I do think the current version of the style will look wrong for a book that has an author and an editor (e.g. a classic work with an editor). How should such a citation look for the journal?

thanks! 

You are right. In the specific case where we have a book with 2 authors and 1 editor, it should look like:

`Hancké, Bob/Rhodes, Martin: Beyond varieties of capitalism: Conflict, contradiction, and complementarities in the European economy, Thatcher, Mark (ed.), Oxford and New York 2007.`

For journals, the editors are not printed (as described in the guidelines [1]).

Is it possible to make this distinction?

[1] https://www.micura.it/images/Ladinia_Redaktionsnormen_Autorenformalia_EN.pdf

File: revista-ladinia.csl [hunk 1]
```diff
@@ -100,13 +100,36 @@
...
+    <group delimiter=" ">
+      <text macro="publisher-place"/>
+      <text macro="issued-year"/>
+    </group>
+  </macro>
+  <macro name="container">
+    <group delimiter=": ">
+    <text term="in"/>
+    <group delimiter=", ">
+      <text variable="container-title" prefix="&quot;" suffix="&quot;"/>
```
[the remark this answers is not in the document]
Comment (schtailmuel) [reply revista-ladinia.csl]: I changed this in the commit [be0b141](https://github.com/citation-style-language/styles/pull/6058/commits/be0b14162931a23e9353fbaa03bdebaa3df5df9e)

**Comment by schtailmuel** [comment]
> You can just add the editor at the appropriate place for book items. If there is no author, the 'substitute' node in the author macro will take care of avoiding duplicate printing

I revised that. Could I kindly ask you to take a look at it again?

**Comment by adam3smith** [comment]
Great, thanks (the change I made corrects the fairly rare case of a book with an author, editor, and translator).

**Pull request merged**